src/quo2/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,296 @@
+(ns quo2.components.wallet.transaction-progress.view
+  (:require [quo2.components.wallet.transaction-progress.style :as style]
+            [quo2.core :as quo2]
+            [quo2.components.selectors.selectors.view :as selectors]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.wallet.progress-bar.view :as progress-box]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.tags.status-tags :as status-tag]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.resources :as resources]
+            [quo2.theme :as theme]
+            [reagent.core :as reagent]
+            [react-native.core :as rn]))
+
+(defn get-colors
+  [color]
+  (let [current-theme (theme/get-theme)]
+    (cond
+      (= current-theme :dark)
+      (cond
+        (= color "neutral-5")  colors/neutral-70
+        (= color "neutral-10") colors/neutral-80
+        (= color "neutral-40") colors/neutral-50
+        (= color "neutral-50") colors/neutral-60
+        (= color "danger-50")  colors/danger-60
+        (= color "success-50") colors/success-60)
+      :else
+      (cond
+        (= color "neutral-5")  colors/neutral-5
+        (= color "neutral-10") colors/neutral-10
+        (= color "neutral-40") colors/neutral-40
+        (= color "neutral-50") colors/neutral-50
+        (= color "danger-50")  colors/danger-50
+        (= color "success-50") colors/success-50))))
```

Comment:
Please use the `colors/custom-color-by-theme` method.

Reply:
imo better to use:
```
(colors/theme-colors <light> <dark> theme)
```

Reply:
this method `get-colors` is very strange imo. it looks much simpler to just the set the colors where they are being used.